components/omega/src/ocn/TracerDefs.inc

```diff
@@ -0,0 +1,42 @@
+//===-- ocn/TracerDefs.inc --------------------*- C++ -*-===//
+//
+/// \file
+/// \brief OMEGA Tracer Definition File
+///
+/// This file defines all OMEGA tracers. It is imported using
+/// the C++ #include preprocessor directive in Tracers.cpp. Only the tracer
+/// definitions selected in the Tracers section of the OMEGA YAML configuration
+/// file will be allocated into memory.
+//===----------------------------------------------------------------------===//
+
+// Index defined for each tracer. The value of these tracer indices are still
+// set in Tracers::init()  and tracers that have not been selected would have
+// standard invalid value: Tracers::IndxInvalid = -1
+inline static I4 IndxTemp             = Tracers::IndxInvalid;
+inline static I4 IndxSalt             = Tracers::IndxInvalid;
+inline static I4 IndxMyBGCTracer      = Tracers::IndxInvalid;
+inline static I4 IndxAnotherBGCTracer = Tracers::IndxInvalid;
+inline static I4 IndxDebug1           = Tracers::IndxInvalid;
+inline static I4 IndxDebug2           = Tracers::IndxInvalid;
+
```

Comment:
remove the index entries that don't have a corresponding define function. That way the index list serves as a list of potentially available tracers.

Reply:
Removed the unused entries.